src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteBuildConfiguration.java

```diff
@@ -1216,28 +1492,44 @@ public DescriptorImpl getDescriptor() {
         return (DescriptorImpl) super.getDescriptor();
     }
 
+    public static DescriptorImpl getDescriptorStatic() {
+        Jenkins jenkins = Jenkins.getInstance();
```

Comment:
Since you need to update to a new Jenkins core anyway, you may want to change it since newer core versions offer methods